index.html

```diff
@@ -8,6 +8,16 @@
 
 <body>
 
+ <h1>Sub Heading 1</h1>
```

Comment:
Please check the design. The first line shows `Heading`. In the code it should be like this:
```suggestion
  <h1>Heading</h1>
```

Because '`<h1>`' is the main heading and then the rest are sub headings. '`<h2>`' is the first sub-heading, etc.